terraform/greenfield-deployment/conditional_standard_only_my_sql.tf

```diff
@@ -31,8 +31,8 @@ resource "azurerm_private_dns_zone" "mysql_zone" {
   
   # Only execute if Standard Tier
   count = (var.skuTier == "Standard" || var.skuTier == "standard" ? 1 : 0)
-
-  name                = "privatelink.mysql.database.azure.com"
+  #name                = "privatelink.mysql.database.azure.com"
```

Comment:
@luisfeliz79, is this commented line necessary?

Reply:
Is not, I can remove it